synfig-studio/src/gui/workarea.cpp

```diff
@@ -1031,12 +1031,23 @@ bool
 WorkArea::on_key_release_event(GdkEventKey* event)
 {
 	SYNFIG_EXCEPTION_GUARD_BEGIN()
+	const float radius((std::fabs(pw)+std::fabs(ph))*4);
```

Comment:
Why is this `radius` defined here, so afar from its real scope?

Reply:
It's being used at 1045 for finding the guide should i move this further down?